data/map.txt

```diff
@@ -29173,6 +29332,35 @@ planet "New Switzerland"
 		threshold 2000
 		fleet "Small Republic" 11
 
+planet "New Tortuga"
+	attributes pirate "core pirate" core religious factory
+	landscape land/fog6
+	description `A planet of violent storms and frequent earthquakes, New Tortuga does not have a lot to offer the average spacefarer. The most significant settlements on the planet are built predominantly underground, far from any fault lines, with only small, hardened, and frequently camoflaged access areas. Combined with the fortress-like blast doors that blockade the spaceport entrances, New Tortuga is the perfect place for those looking for somewhere to hide.`
```

Comment:
```suggestion
	description `A planet of violent storms and frequent earthquakes, New Tortuga does not have a lot to offer the average spacefarer. The most significant settlements on the planet are built predominantly underground, far from any fault lines, with only obscure, hardened access areas. Combined with the fortress-like blast doors that blockade the spaceport entrances, New Tortuga is the perfect place for those looking for somewhere to hide.`
```
![image](https://user-images.githubusercontent.com/4471575/177924172-d64cfded-2e4d-4890-9660-366d6ea46b03.png)
This should get it cleanly fitting in the description frame.